Scripts/NuGet/EdFi.RestApi.Databases/Deployment.psm1

```diff
@@ -103,10 +103,11 @@ function Initialize-DeploymentEnvironment {
     if ($OdsTokens) { $settings.ApiSettings.OdsTokens = $OdsTokens }
     if ($OdsDatabaseTemplateName) { $settings.ApiSettings.OdsDatabaseTemplateName = $OdsDatabaseTemplateName }
     if ($DropDatabases.IsPresent) { $settings.ApiSettings.DropDatabases = $DropDatabases.IsPresent }
-    if ($UsePlugins.IsPresent) { $settings = (Merge-Hashtables $settings, (Get-EdFiDeveloperPluginSettings)) }
 
     Set-DeploymentSettings $settings | Out-Null
 
+    if ($UsePlugins.IsPresent) { $settings = (Merge-Hashtables $settings, (Get-EdFiDeveloperPluginSettings)) }
+
```

Comment:
so it looks like passing in `-useplugins` here has no effect because, unlike initdev where these settings are being saved to the appsettings.development.json, when deploying these settings only storage is in the cache. There is no mechanism for deployment to save to a file at the moment, and user-secrets are not supported. If we revert the changes to this file it should work again, but users would have to restart their powershell session to clear the cache when using the `-useplugins` parameter.

cc @vimayya

Reply:
@tbmeador-dlp restarting their powershell session to clear the cache when switching the -useplugins parameter is alright.